planning/behavior_path_planner/autoware_behavior_path_planner_common/include/autoware/behavior_path_planner_common/utils/path_safety_checker/path_safety_checker_parameters.hpp

```diff
@@ -75,22 +78,30 @@ struct PredictedPathWithPolygon
 struct ExtendedPredictedObject
 {
   unique_identifier_msgs::msg::UUID uuid;
-  geometry_msgs::msg::PoseWithCovariance initial_pose;
-  geometry_msgs::msg::TwistWithCovariance initial_twist;
-  geometry_msgs::msg::AccelWithCovariance initial_acceleration;
-  autoware_perception_msgs::msg::Shape shape;
-  std::vector<autoware_perception_msgs::msg::ObjectClassification> classification;
+  Pose initial_pose;
+  Twist initial_twist;
+  Shape shape;
+  ObjectClassification classification;
+  Polygon2d initial_polygon;
   std::vector<PredictedPathWithPolygon> predicted_paths;
+  double dist_from_ego{0.0};
```

Comment:
@zulfaqar-azmi-t4
Can you add descriptions such as the distance from where to where?
I'm asking this without fully understanding the structure, but is this variable expected to be updated every cycle?
Also, this member variable doesn't seem to be used anywhere. Is it necessary for lane changes?

Reply:
@kyoichi-sugahara 

> Can you add descriptions such as the distance from where to where?
Sure.

> I'm asking this without fully understanding the structure, but is this variable expected to be updated every cycle?
Yes, ExtendedPredictedObject is updated every cycle. 

> Also, this member variable doesn't seem to be used anywhere. Is it necessary for lane changes?
Yes, lane change  is using them. Again, I didn't want to put the changes in this PR.

Reply:
Thank you for the reply!
If that's the case, it would be good👍 I wanted to avoid discrepancies between the values and reality by ensuring that the information is not outdated.

Reply:
added comment for distance 8a1614f0c04ef06751ab5daeb2e2dd7d2d98cfe7